File: V семестр/Паралельне програмування/Кочетов/лабы/lab2/src/F3.java
public class F3 extends Thread {
    private Vector result;
    public Vector getResult() {
        return result;
    }

    private int N;

    F3(String name, int priority, int N) {
        setName(name);
        setPriority(priority);
        this.N = N;
    }

    @Override
    public void run() {
        try {
            sleep(100);
            System.out.println("Task 3 start");
            Vector S = new Vector(N);
            Matrix MO = new Matrix(N), MP = new Matrix(N), MR = new Matrix(N);
            result = MO.multiply(MP).multiply(S).sum(MR.multiply(S.sort()));
            System.out.println("Task 3 end");
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
}

File: V семестр/Паралельне програмування/Бровченко/lab3_OPP/lab3_OPP/lab3_OPP/F3.cs
using System;

namespace lab3
{
    public class F3
    {
    private Vector result;
    public Vector getResult()
    {
        return result;
    }

    private int N;

    public F3(int N)
    {
        this.N = N;
    }

    public void run()
    {
            Console.WriteLine("Task 3 start");
            Vector P = new Vector(N);
            Matrix MT = new Matrix(N), MR = new Matrix(N);
            result = MR.multiply(MT).multiply(P.sort());
            Console.WriteLine("Task 3 end");
    }
}

}
